staging/src/k8s.io/client-go/OWNERS

```diff
@@ -8,6 +8,7 @@ approvers:
   - smarterclayton
   - sttts
   - yliaog
+  - apelisse
```

Comment:
Similar here, can we reviewer first?

Also, thinking about the future.  The combination of apimachinery and client-go are heavily impacted by code-generator and vice-versa.  It's not particularly attractive, but I think requiring stewardship of the code-generator to maintain apimachinery and client-go seems like a good idea.

Reply:
Yeah, that makes sense, I'll add reviewer. Would it make sense to add approval to `staging/src/k8s.io/client-go/discovery`, `staging/src/k8s.io/client-go/applyconfigurations`, `staging/src/k8s.io/client-go/openapi`?

Reply:
+1 for openapi, that sounds fine, I don't think anyone else knows better

Reply:
> 2. staging/src/k8s.io/client-go/discovery - Do you have changes in mind or just maintenance? If you're asking because you want to make changes, I'd like to make the changes first and then hand over the keys. If you're thinking about maintenance plus aggregated discovery, I'd like to talk about desired outcomes, but probably yes.

I don't have changes planned, but let's do it once aggregated is beta.

Reply:
> I don't have changes planned, but let's do it once aggregated is beta.

That sounds good.